src/benchmarks/micro/libraries/System.Diagnostics/Perf_Activity_Enumerations.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+
+namespace System.Diagnostics
+{
+    [MemoryDiagnoser]
```

Comment:
nit: `MemoryDiagnoser` is enabled by default in this repo, there is no need for the attribute  here

https://github.com/dotnet/performance/blob/7d9d860af8cfd8807975517015ca6d9f98985852/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs#L43

```suggestion
```

Reply:
/cc @tarekgh